metadata.yaml

```
@@ -18,7 +18,7 @@ resources:
   oci-image:
     type: oci-image
     description: Backing OCI image
-    upstream-source: charmedkubeflow/pvcviewer-controller:1.10.0-rc.1-3a8beff
+    upstream-source: charmedkubeflow/pvcviewer-controller:1.10.0-09b616ff
```

looks like there's a typo in the tag
```suggestion
    upstream-source: charmedkubeflow/pvcviewer-controller:1.10.0-09b616f
```